### DISCUSSION TITLE
如何给Form设置自定义的错误提示信息？

### DISCUSSION THREAD

**Discussion by SyuTingSong** [ant-design/ant-design-mobile#5496] [General]
前端检查都过了，但总有一些检查工作是服务端执行的，比如注册时账号已经存在，还有填写的内容没有通过敏感词检查等等。现在服务端返回了errors对象，提供了服务端检查没通过的表单字段名和对应的错误信息，如何才能把这些消息设置到Form组件中，让它们显示在对应字段的Form.Item错误提示区？

**Comment by IronKinoko** [comment]
https://codesandbox.io/s/sweet-jasper-hpvdiz?file=/src/App.tsx

`form.setFields` 可以实现你的需求，具体实现看上面的链接

**Reply by SyuTingSong** [reply]
试了有用，非常感谢。

不过还想了解是否有声明式的方法可以做到，显示错误消息的地方是否有对应的组件props可以设置

**Reply by IronKinoko** [reply]
目前没有声明式的方法。`Form` 组件内部是使用 [`rc-field-form`](https://github.com/react-component/field-form) 实现的，可以查看对应的文档
